unstract/connectors/src/unstract/connectors/databases/bigquery/bigquery.py

```diff
@@ -18,11 +18,14 @@
 
 logger = logging.getLogger(__name__)
 
+BIG_QUERY_TABLE_SIZE = 3
```

Comment:
Can you add a small inline comment or anything if possible ,to undesrtand what it means?